packages/maskbook/src/social-network-adaptor/twitter.com/collecting/post.ts

```diff
@@ -47,7 +47,12 @@ function registerPostCollectorInner(
     const watcher = new IntervalWatcher(postsContentSelector())
         .useForeach((node, _, proxy) => {
             const tweetNode = getTweetNode(node)
-            if (!tweetNode) return
+            if (!tweetNode) {
+                if (process.env.NODE_ENV === 'production') {
+                    console.log('Fails to get tweetNode from node', node)
```

Comment:
Why do you log it in production, not in dev?

Reply:
Oops, that's a typo. Fixed

Reply:
why not is `NODE_ENV !== 'production'`?